cards/eye-ddx/card.md

```diff
@@ -0,0 +1,33 @@
+---
+
+title: Rule of 2s for Ocular Emergencies
+author:
+    - Puja Gopal, MD
+created: 2017/04/14
+updates: null
+categories:
+    - Ophthalmology
+ 
+---
+# Background
+
+Eye complaints are a common chief complaint in the emergency department (ED). Approximately 3.3% of patient visits to the ED were related to an eye or ear complaint in 2005. Having a systematic approach to ocular complaints is especially important as each structure of the eye holds within it different pathology. It is often difficult to remember individual diagnoses for various complaints such as red eye, painful eye, or vision loss, especially during busy shifts. The rule of 2s, as illustrated below, is a simple method with an anatomical approach to major ocular diagnoses encountered in the ED. It ensures one doesn't miss clinically significant diagnoses and requires only a basic understanding of the anatomy of the eye. The rule of 2s is easy to remember as it contains the number 2 in the context of eyes and proceeds from layer to layer of the eye in an anterior to posterior direction highlighting the 2 major associated diagnoses of that layer.  

```

Comment:
I was thinking about it before submitting this new version, but it seemed to lose the flow if I tried. If you have any suggestions as to at what point I should, will definitely do it!